flink-container/kubernetes/job-cluster-job.yaml.template

```diff
@@ -31,7 +31,7 @@ spec:
       containers:
       - name: flink-job-cluster
         image: ${FLINK_IMAGE_NAME}
-        args: ["job-cluster", "-Djobmanager.rpc.address=flink-job-cluster",
+        args: ["job-cluster", "--job-classname=${JOB_CLASSNAME}", "-Djobmanager.rpc.address=flink-job-cluster",
```

Comment:
I don't think we always want to set the job class name here. We scan the class path for a JAR with a main class manifest entry. Therefore, the job cluster entry point should figure out automatically which class to run in the common case.

Reply:
I think It write into docment will more suitable.

Reply:
I know how to scan the main class, I think we need a more suitable methed to tell user to add the main class.

Reply:
How is an empty string for `--job-classname` handled by the main method? If it's ignored by choice and falls back to class path scanning, this could be ok. Otherwise, I still think that making this required would hurt common case user experience.